pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.sh

```diff
@@ -60,10 +59,6 @@ while [ "$#" -gt 0 ]; do
         upgrade=1
         upgrade_all=1
         ;;
-      --repair)
-        repair=1
-        extraBuildFlags+=("$i")
```

Comment:
Doesn't this mean that `--repair` would have been passed through to nix-build, which does have a `--repair` option?  The `repair` variable might not have been used, but it looks like the option did something.

Reply:
No, look below, it still get's captured.

My guess is that someone wanted to do some special logic around this argument specifically, but has since been removed